h/schemas/forms/admin/group.py

```diff
@@ -117,8 +121,8 @@ def validate(node, value):
 @colander.deferred
 def group_organization_select_widget(_node, kwargs):
     orgs = kwargs["organizations"]
-    org_labels = []
-    org_pubids = []
+    org_labels = ["-- None --"]
+    org_pubids = [""]
```

Comment:
Add a `-- None --` option to the organization dropdown (`<select>`) on the admin page for creating or editing an open or restricted group.